control-plane/roles/monitoring/templates/gardener-metrics-exporter/deployment-metrics-exporter.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
Skip gardener for this PR.

Reply:
https://github.com/metal-stack/metal-roles/commit/99180ee478459d677e388080ad8e1df35d50557f